sim/battle-stream.ts

```diff
@@ -41,12 +41,14 @@ function splitFirst(str: string, delimiter: string, limit: number = 1) {
 export class BattleStream extends Streams.ObjectReadWriteStream<string> {
 	readonly debug: boolean;
 	readonly keepAlive: boolean;
+	readonly retainBattle: boolean;
```

Comment:
We can just always retain the battle, instead of needing a setting for it.

Reply:
Like, the only reason we don't is because a past V8 had GC issues which we thought preemptively deleting references might help fix.

Reply:
Gotcha.